bindings/java/src/main/com/apple/foundationdb/FDBTransaction.java

```diff
@@ -223,6 +242,9 @@ public void setReadVersion(long version) {
 	 */
 	@Override
 	public CompletableFuture<Long> getReadVersion() {
+		if (eventKeeper != null) {
+			eventKeeper.increment(Events.JNI_CALL);
```

Comment:
it's probably useful to remember that the set of KPIs that I used here are more-or-less arbitrary and are almost certainly not complete--the likelihood is high that we'll want other metrics over time as we need them. 

Tracking JNI calls is really more of a debugging than a monitoring step (since there's not anything operational that can be done about it), but it's often proved useful to me in the past, so I wanted to include it.